src/Modifier/StreamIterator.php

```diff
@@ -156,18 +156,17 @@ public function setFlags($flags)
      * @param array  $fields
      * @param string $delimiter
      * @param string $enclosure
-     * @param string $escape
      *
      * @return int
      */
-    public function fputcsv(array $fields, $delimiter = null, $enclosure = null, $escape = null)
+    public function fputcsv(array $fields, $delimiter = ',', $enclosure = '"', $escape = '\\')
```

Comment:
StreamIterator is marked as `@internal` so anyone using it in public API exposes himself to harm.